### PR TITLE
Add some methods and fix crash at startup

### PR DESCRIPTION
**Pull request by tommybuonomo**
Add the 2 methods disableLocationPermissionCheck and disableWriteExternalStoragePermissionCheck (cf https://wiki.appodeal.com/en/android/2-7-4-android-sdk-integration-guide/configure-sdk)

Fix a crash at startup because of illegal argument on the error response of the plugin. If the user starts the app without connection (no WIFI, no 4G) the app was crashing because of that on the method `fetchConsentInfo`

### CODE REVIEW

**Comment by vegidio** [comment]
@tommybuonomo Thanks for the PR. I will review it over the weekend!

File: android/src/main/kotlin/io/vinicius/appodeal_flutter/AppodealFlutterPlugin.kt
Comment (vegidio) [inline android/src/main/kotlin/io/vinicius/appodeal_flutter/AppodealFlutterPlugin.kt:5]: Can you please use individual imports instead of wildcard imports?

I know this is one of those debates, like tabs vs spaces, and in my opinion there's no specific advantage or disadvantage to use on method over the other, but since I adopted the code style in the project to import each class individually, then I would like to stick with this code style in all files in the project 🙂

File: lib/src/appodeal.dart
Comment (vegidio) [inline lib/src/appodeal.dart:195]: I see that you did this change for the Android integration only, but do you also have experience with iOS? If yes, then it would great if you could do this change on the iOS code base as well so both platforms are supported 🙂

**Review by vegidio — CHANGES_REQUESTED** [review]
The PR is looking very good to me, so thanks for the contribution! But I l suggested two changes there and it would be great if you work on them before I merge.

**Comment by houdayec** [comment]
@vegidio @tommybuonomo Since this is a very important fix (otherwise, package is not usable in production), could you try to finish this PR? Also, @vegidio , I don't see any updates since a month ago, did you drop the package? I know it requires time to maintain, I just want to know.

**Comment by vegidio** [comment]
@houdayec I will gladly merge the PR as soon as I hear something from @tommybuonomo with the comments that I put in his change.

As for the package itself, it indeed doesn't have any change for quite some time, but it's not abandoned yet. I just need to focus on my primary job right now so unfortunately I cannot dedicate my time to this or any other personal project that I have. I believe I will have time to work on this project again in the beginning of January.

Meanwhile any PRs are welcomed.

**Comment by houdayec** [comment]
> @houdayec I will gladly merge the PR as soon as I hear something from @tommybuonomo with the comments that I put in his change.
> 
> As for the package itself, it indeed doesn't have any change for quite some time, but it's not abandoned yet. I just need to focus on my primary job right now so unfortunately I cannot dedicate my time to this or any other personal project that I have. I believe I will have time to work on this project again in the beginning of January.
> 
> Meanwhile any PRs are welcomed.

Thanks for your reply. You did a very great job so far! I am looking for such a package for months now, started to do my own on a side project but yours is definitely a lot more advanced. I am waiting for this PR to be merged (maybe I will take @tommybuonomo PR , make the changes and submit PR as he does not respond). I will also make some tests tomorrow to check if the package do not crash on iOS when phone is not connected to the Internet. I'm ready to help you as contributor on this one, keep going bud.

**Comment by vegidio** [comment]
@houdayec Thanks for helping with this! Looking forward for your changes 🙂

**Comment by tommybuonomo** [comment]
Hey guys, 
Sorry for the late, my primary job kept me busy during the last weeks.
I'll do the changes today about the PR, and I'll check the behaviour on iOS :)

**Comment by houdayec** [comment]
> Hey guys,
> Sorry for the late, my primary job kept me busy during the last weeks.
> I'll do the changes today about the PR, and I'll check the behaviour on iOS :)

Thanks for your reply. Let us know!

**Comment by tommybuonomo** [comment]
@vegidio @houdayec Changes done, let me know if it's ok for you !
I also added a new method on iOS to disable location tracking.
Thanks

**Comment by vegidio** [comment]
@tommybuonomo Thanks for the changes! I will have a look either today or tomorrow and merge asap 🙂

**Review by vegidio — APPROVED** [review]

**Comment by vegidio** [comment]
@tommybuonomo I just merged your PR. Thank you very much for the contribution. I will generate a new package and upload it to pub.dev later today 🏆

**Comment by vegidio** [comment]
@tommybuonomo @houdayec a new package was published on pub.dev. I bumped it to version `0.4.0` 🚀

However, while I was trying to use the newest version of the package I got an error regarding Appodeal's TapJoy integration on iOS. This doesn't seem to be related to the changes that @tommybuonomo did and it's probably going to disappear when we bump the Appodeal plugin to the newest version.

There's already another PR #14 to upgrade Appodeal for the newest version so I will try to review it ASAP. So in case version `0.4.0` doesn't work yet for you on iOS due to this TapJoy integration, just wait a little more for this other PR 🙂

**Pull request merged**
